newIDE/app/src/SceneEditor/index.js

```diff
@@ -1240,6 +1240,7 @@ export default class SceneEditor extends React.Component<Props, State> {
       multiSelect: true,
       layersVisibility: null,
     });
+    this.forceUpdatePropertiesEditor();
```

Comment:
should this be part of the `onInstanceAdded()` ?
If not, do you need to add it in the `_addInstance()` too ?

Reply:
To me, onInstanceAdded is used to perform additional work following the addition of an instance (display an info bar for example). The logic in panels should be handled by those functions.
_addInstance does not select the instance after it's been added so no need to force update the properties panel.